src/OnboardingSPA/components/TextInput/TextInputSiteGen/stylesheet.scss

```diff
@@ -48,8 +48,7 @@ $selected-detail: #1de082;
 
 		&_bottom {
 			display: flex;
-			flex-direction: row;
-			align-items: center;
+			flex-direction: column;
```

Comment:
this seems like a rather big layout change to this component, are we sure this is not impacting anything else anywhere else, maybe in DIY flow?

Reply:
no need to worry this only targets elements under this CSS class .nfd-sg-input .. it will not affect the DIY or any other part of onboarding